maquetado/scss/login.scss

```diff
@@ -0,0 +1,94 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wolox-image-login {
+  align-self: center;
+  height: 70px;
+  margin-top: 20px;
+  width: 200px;
+}
+
+.login-container {
+  align-items: center;
+  background-color: $light-grey;
+  border-top: 10px solid $light-blue;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: flex-start;
+  margin-top: 70px;
+  width: 300px;
+  max-width: 300px;
+}
+
+.label {
+  align-self: flex-start;
+  color: $black;
+  font-family: Roboto;
+  font-size: 15px;
+  height: 20px;
+  line-height: 25px;
+  margin-bottom: 5px;
+  margin-left: 25px;
+  margin-top: 10px;
+  width: 40px;
```

Comment:
If I am not mistaken I think you don't need to set a width here. The content will be set the width of every label